### PR TITLE
[15.0][MIG] account_balance_line

### PR DESCRIPTION
**Pull request by koenloodts**

### CODE REVIEW

File: account_balance_line/README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/14.0/account_balance_line
```
Comment (Shide) [inline account_balance_line/README.rst:17]: Search for references to branch 14.0 and change it to 15.0
Non blocker

**Review by Shide — APPROVED** [review]
LGTM

**Comment by rafaelbn** [comment]
Hello @koenloodts , please could you change the 14.0 to 15.0 in all the references? Thanks

**Review by rafaelbn — APPROVED** [review]
Perfect! Thank you! 👍

**Comment by rafaelbn** [comment]
@HaraldPanten please this is an easy one 😄 🙏

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Seems that tests are failing. Could you rebase to check?

THX

**Comment by HaraldPanten** [comment]
@koenloodts

**Comment by rafaelbn** [comment]
Please @koenloodts could you review the test message?

```
Run oca_install_addons
+ setuptools-odoo-get-requirements --include-addons --addons-dir .
/root/.local/pipx/shared/lib/python3.8/site-packages/_distutils_hack/__init__.py:17: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/root/.local/pipx/shared/lib/python3.8/site-packages/_distutils_hack/__init__.py:30: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
+ cat test-requirements.txt
+ pip install -r test-requirements.txt
odoo-addon-account_move_line_stock_info>=15.0dev,<15.1dev
odoo-addon-product_category_product_link>=15.0dev,<15.1dev
odoo>=15.0a,<15.1dev
vatnumber
WARNING: The directory '/github/home/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
Looking in indexes: https://wheelhouse.odoo-community.org/oca-simple-and-pypi
Collecting odoo-addon-account_move_line_stock_info<15.1dev,>=15.0dev
  Downloading odoo_addon_account_move_line_stock_info-15.0.1.0.0.3-py3-none-any.whl (59 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 59.4/59.4 KB 80.2 MB/s eta 0:00:00
Collecting odoo-addon-product_category_product_link<15.1dev,>=15.0dev
  Downloading odoo_addon_product_category_product_link-15.0.1.0.0.2-py3-none-any.whl (20 kB)
Requirement already satisfied: odoo<15.1dev,>=15.0a in /opt/odoo (from -r test-requirements.txt (line 3)) (15.0)
Collecting vatnumber
  Downloading vatnumber-1.2.tar.gz (19 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in vatnumber setup command: use_2to3 is invalid.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Error: Process completed with exit code 1.
```

**Comment by rafaelbn** [comment]
@Yajo please could you help here? Thank you!

MT-348

**Comment by rafaelbn** [comment]
Hello @koenloodts , thanks for this migration. We have ping you and you don't answer. Are you going to continue this PR?

**Pull request merged**